webui/ocw/apps.py

```diff
@@ -1,5 +1,24 @@
 from django.apps import AppConfig
+import os
+import logging
+
+logger = logging.getLogger(__name__)
 
 
 class OcwConfig(AppConfig):
     name = 'ocw'
+    ready_called = False
```

Comment:
is it "ready for call" ? or most probably "already called" ?

Reply:
now I get naming ... most suitable I would say will be `init_cron_called` for variable and `init_cron` for function . WDYT ?

Reply:
Look here https://docs.djangoproject.com/en/2.2/ref/applications/#django.apps.AppConfig.ready it means that the `ready()` function was called. 
So maybe `ready_fn_called` ?

Reply:
ah I was not aware that you inherit method name . but from what I have read you don't need this variable at all .  https://docs.djangoproject.com/en/2.2/ref/applications/#django.apps.AppConfig.ready clearly saying that method will be called **only** in initialization phase , right ?

Reply:
There is a note
```
.... , ready might be called more than once. ....
```

Reply:
> There is a note
> 
> ```
> .... , ready might be called more than once. ....
> ```

ok I see . let's just stick with name which you initially set :) thanks for clarifications ...